src/plugins/blocksmith/components/article-block-modal/CustomizeOptions.jsx

```diff
@@ -68,6 +71,11 @@ const CustomizeOptions = ({
         <Checkbox size="sd" id="hasUpdatedDate" checked={hasUpdatedDate} onChange={handleChangeCheckbox}>
           Is update
         </Checkbox>
+        <Checkbox size="sd" id="aside" checked={aside} onChange={handleChangeCheckbox} className="ml-3">
+          Aside
+        </Checkbox>
+        <Icon imgSrc="info-outline" id="aside-tooltip" size="xs" style={{ marginLeft: '0.3rem' }} />
+        <UncontrolledTooltip key="tooltip" placement="bottom" target="aside-tooltip">Is only indirectly related to the main content.</UncontrolledTooltip>
```

Comment:
@nanetteCodes this does not need a key. same for the others. also arguably doesnt need the placement unless you think bottom works best as opposed to the default `auto`